install/kubernetes/helm/istio/values-e2e.yaml

```diff
@@ -0,0 +1,41 @@
+# This file overrides values for e2e testing.
+
+global:
+  proxy:
+    concurrency: 0
+    resources:
+      requests:
+        cpu: 10m
+        memory: 40Mi
+
+    accessLogFile: "/dev/stdout"
+    enableCoreDump: true
+
+  disablePolicyChecks: false
+
+prometheus:
+  scrapeInterval: 5s
+
+gateways:
+  istio-ingressgateway:
+    autoscaleMax: 1
+
+mixer:
+  policy:
+    enabled: true
+    replicaCount: 2
+    autoscaleEnabled: false
+  telemetry:
+    enabled: true
+    loadshedding:
+      mode: disabled
+  adapters:
+    stdio:
+      enabled: true
+  resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 100m
+        memory: 100Mi
```

Comment:
Since this file is introduced, can you make sure it is not packaged in release binary?  I think there is a file that you exclude the list of values not in release.